src/hammer-vlsi/technology/sky130/__init__.py

```diff
@@ -264,5 +269,30 @@ def sky130_power_nets(ht: HammerTool) -> bool:
                 ht.verbose_append("connect_global_net {tie} -type net -net_base_name {net}".format(tie=pwr_gnd_net.tie, net=pwr_gnd_net.name))
     return True
 
-tech = SKY130Tech()
+def sky130_connect_nets(ht: HammerTool) -> bool:
+    assert isinstance(
```

Comment:
You also need an assertion that `ht` is a `TCLTool` to get rid of the type checker failures against `append` and `verbose_append`.